core/storage/user/gae_models.py

```diff
@@ -168,6 +175,17 @@ def is_normalized_username_taken(cls, normalized_username):
         return bool(cls.get_all().filter(
             cls.normalized_username == normalized_username).get())
 
+    @classmethod
+    def get_by_gae_id(cls, gae_id):
+        """Returns a user model with given GAE user ID.
+        Args:
```

Comment:
Follow existing docstring format: leave one line above Args and one line above Returns.